Readme.md

```diff
@@ -38,6 +38,7 @@ This is the <a href="https://code.visualstudio.com/">Visual Studio Code</a> exte
 - Use a specific Runtime provider for the Camel catalog.
 - Additional Camel components can be provided.
 - Use dashed notation in properties file and Camel K modelines.
+- Automatic schema association for Camel YAML file
```

Comment:
not sure it is worthy to mention it in the "feature" list. This is not a feature provided by this extension at all.

Reply:
agreed